storage/tianmu/core/temp_table.h

```diff
@@ -391,6 +391,8 @@ class TempTable : public JustATable {
 
   void MarkCondPush() { can_cond_push_down = true; };
   bool CanCondPushDown() { return can_cond_push_down; };
+  std::string GetTableName() override { return std::string("-"); }
```

Comment:
what is it?

Reply:
Polymorphic implementation is compatible with temporary tables

Reply:
> Polymorphic implementation is compatible with temporary tables

does not make sense.